volume/volinit.f

```diff
@@ -530,6 +530,8 @@ SUBROUTINE VOLINIT(REGN,FORST,VOLEQ,MTOPP,MTOPS,STUMP,DBHOB,
       IF(STUMP.LE.0) STUMP = 1.0
 C  calc stump DIB
       HTUP = STUMP
+C  this line traps cases where the calls to calcdia2 do not return a DIB.
+      DIB = -1.0
```

Comment:
Issue is addressed in seperate upcoming commit, initializing DIB to 0, as well as initializing other variables and updates to this file